bulk-print-fail-alert.tf

```diff
@@ -0,0 +1,16 @@
+module "cmc-bulk-print-fail-alert" {
+  source = "git@github.com:hmcts/moj-module-metric-alert"
+
+  app_insights_name = "${var.env == "prod" ? "cmc-prod" : "cmc-sandbox"}"
```

Comment:
its just cmc-${env}

Reply:
I'd seen there are some environments without a corresponding application insights resource and somehow came to the conclusion that anything other than `prod` would work against `sandbox`... There's no `cmc-saat` application insights resource, for example. Confirm?

Reply:
there's 
cmc-prod
cmc-demo
cmc-aat
cmc-sandbox
cmc-saat 
cmc-sprod

Some of which may have been deleted at some point
but it is cmc-${var.env}

Reply:
addressed